src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -165,8 +169,12 @@ launchNotifier nat =
         , updateNotifier
         ]
   where
-    cooldownPeriod = 5000000         -- 5 sec
+    cooldownPeriod :: Microsecond
+    cooldownPeriod = 5000000
```

Comment:
Agree, it was. Because this function used `threadDelay` from `base` with `Integer` as time. So comment could clarify things. Now with explicitly specified time units everyone could see this is exactly `5 sec`. There was no reason to remove this comment intentionally, I just did it accidentally. But I don't think there are much sense in adding it back again.